### PR TITLE
Gui 18 migration on ts and angular

### PR DESCRIPTION
**Pull request by AndreyKoltsov1997**
Changing UI structure to be compatible with Angular 6. 
It completely deletes deprecated JS code.

### CODE REVIEW

File: deprecated/lib/META-INF/maven/org.webjars/bootstrap/pom.xml [hunk 1]
```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (veronikaKochugova) [inline deprecated/lib/META-INF/maven/org.webjars/bootstrap/pom.xml]: Why pom.xml??? We use Gradle

Comment (veronikaKochugova) [inline deprecated/lib/META-INF/maven/org.webjars/bootstrap/pom.xml]: Why pom.xml??? We use Gradle

**Comment by akurilov** [comment]
It's already 10K of new lines of code instead of 110K. Much better, but still quite hard to review :)

**Comment by akurilov** [comment]
- Not ready actually, see the conflicts
- Tried to start, got the following:
``
docker-compose up
Pulling mangoose-console (emcmongoose/mongoose-console:0.1.1)...
0.1.1: Pulling from emcmongoose/mongoose-console
4fe2ade4980c: Already exists
e11b023b07a0: Pull complete
aaf7d5908de5: Pull complete
8ae3a5ca453f: Pull complete
fe19fb8e33e6: Pull complete
1781b162cdb2: Pull complete
Digest: sha256:5c2603910cdc6b75142e6356d63fcea68f1f31747465265dc73711a44163e9af
Status: Downloaded newer image for emcmongoose/mongoose-console:0.1.1
Recreating console_mangoose-console_1 ... 
Starting console_grafana_1            ... 
Starting console_prometheus_1         ... error

Starting console_grafana_1            ... error
dbbcc8b91887329f634108e6fe9be7356700/merged: stale NFS file handle
Recreating console_mangoose-console_1 ... done
ERROR: for console_grafana_1  Cannot start service grafana: error creating overlay mount to /home/docker/overlay2/7be9f8fc7efa48695ea3b6bcbbdee0b38defc08cb43a49a2313a83cd81a6830a/merged: stale NFS file handle

ERROR: for prometheus  Cannot start service prometheus: error creating overlay mount to /home/docker/overlay2/7e9bb5634c029c34bd1b06be751edbbcc8b91887329f634108e6fe9be7356700/merged: stale NFS file handle

ERROR: for grafana  Cannot start service grafana: error creating overlay mount to /home/docker/overlay2/7be9f8fc7efa48695ea3b6bcbbdee0b38defc08cb43a49a2313a83cd81a6830a/merged: stale NFS file handle
ERROR: Encountered errors while bringing up the project.
``

**Comment by veronikaKochugova** [comment]
@akurilov I have not such problem
````
[user@veronika console]$ docker-compose up
Starting console_grafana_1_f73a3cba937d          ... done
Starting console_prometheus_1_dad433c2d1f0       ... done
Starting console_mangoose-console_1_2c71fc9f4955 ... done
Attaching to console_grafana_1_f73a3cba937d, console_prometheus_1_dad433c2d1f0, console_mangoose-console_1_2c71fc9f4955
````

**Pull request merged**